tools/jit/gen_jit_dispatch.py

```diff
@@ -481,6 +481,8 @@ def format_arg(arg):
     arg_list = ', '.join(args)
     if len(decl['returns']) == 1:
         ret_list = jit_type_of(decl['returns'][0])
+        if 'field_name' in decl['returns'][0] and decl['returns'][0]['field_name']:
```

Comment:
I made this comment in the other PR (maybe I didn't post it yet), but you can simplify this to `if decl['returns'][0].get('field_name')`.  Also a comment such as "add output name to signature" would be great.